operator/pkg/ciliumidentity/controller_test.go

```diff
@@ -414,31 +405,24 @@ func TestUpdateUsedCIDIsReverted(t *testing.T) {
 		t.Fatalf("update CID: %v", err)
 	}
 
-	// Wait for update event to propagate.
-	ev := <-events
-	if ev.Kind != resource.Upsert {
-		t.Fatalf("expected upsert event, got %v", ev.Kind)
-	}
-	if ev.Key.Name != toUpdate.Name {
-		t.Fatalf("expected upsert event for cid %s, got %v", toUpdate.Name, ev.Key.Name)
-	}
-	if !cmp.Equal(ev.Object.SecurityLabels, updated.SecurityLabels) {
-		t.Fatalf("expected labels %v, got %v", updated.SecurityLabels, ev.Object.SecurityLabels)
+	cid, err := fakeClient.CiliumV2().CiliumIdentities().Get(ctx, updated.Name, metav1.GetOptions{})
```

Comment:
Let's add the check for the returned `err`

Reply:
done